test/tests/response.test.js

```diff
@@ -26,10 +26,10 @@ describe('Handing server responses', () => {
         done()
       })
   })
-  it('Route not matched, respond with a 404', (done) => {
+  it('Route not matched, respond with a 200', (done) => {
     request
       .get(`${tapestry.server.info.uri}/route/not/matched/in/any/way`, (err, res) => {
-        expect(res.statusCode).to.equal(404)
+        expect(res.statusCode).to.equal(200)
```

Comment:
200 feels funny. 

```
  // You can also check renderProps.components or renderProps.routes for
  // your "not found" component or route respectively, and send a 404 as
  // below, if you're using a catch-all route.
```

If we named the last default route "not-found" or something along those lines.

On the server could we do `if (renderProps.routes[0].name === 'not-found') let status = 404` ?